ui/components/ui/callout/callout.stories.js

```diff
@@ -81,7 +81,7 @@ export const MultipleDismissibleCallouts = () => {
   };
 
   return (
-    <Box borderColor={COLORS.BORDER_DEFAULT} padding={[8, 0, 0, 0]}>
+    <Box borderColor={COLORS.BORDER_DEFAULT} paddingTop={8}>
       <Box margin={2}>
         <Typography variant={TYPOGRAPHY.H4}>
           This is your private key:
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props